app/src/main/java/com/sun/makeupwindow/ui/home/HomeFragment.kt

```diff
@@ -125,6 +126,10 @@ class HomeFragment : BaseFragment(), HomeContract.View {
     }
 
     private fun itemCategoryClicked(category: Category) {
+        fragmentManager?.addFragment(
+            R.id.frameContainer,
+            ProductTypeFragment.getInstance(category)
```

Comment:
sao fragment lại getInstance?

Reply:
anh có thể cho em được tại sao không dùng getInstance đối với fragment không ạ

Reply:
à, bên trong companion object của em là 
companion object {
        private const val BUNDLE_PRODUCT_DETAIL = "BUNDLE_PRODUCT_DETAIL"

        fun getInstance(product: Product) = ProductDetailFragment().apply {
            arguments = bundleOf(BUNDLE_PRODUCT_DETAIL to product)
        }
    }

Reply:
em sẽ đổi tên lại là newInstance